common/darwin/Classes/FlutterRTCMediaStream.m

```diff
@@ -490,10 +492,22 @@ -(void)getDisplayMedia:(NSDictionary *)constraints
     RTCMediaStream *mediaStream = [self.peerConnectionFactory mediaStreamWithStreamId:mediaStreamId];
 
     RTCVideoSource *videoSource = [self.peerConnectionFactory videoSource];
-    FlutterRPScreenRecorder *screenCapturer = [[FlutterRPScreenRecorder alloc] initWithDelegate:videoSource];
```

Comment:
Should we have an option for using the in app capturer vs. broadcast upload extension capturer?